src/commands/dev/lib/builder-cache.ts

```diff
@@ -31,7 +31,8 @@ export const builderDirPromise = prepareBuilderDir();
  * Prepare cache directory for installing now-builders
  */
 export async function prepareCacheDir() {
-  const designated = cacheDirectory('co.zeit.now');
+  const { NOW_BUILDER_CACHE_DIR } = process.env;
+  const designated = NOW_BUILDER_CACHE_DIR || cacheDirectory('co.zeit.now');
```

Comment:
Should we call `path.resolve` on `NOW_BUILDER_CACHE_DIR`?

Reply:
Good call, updated